etc/copyright-exclude.txt

```diff
@@ -66,3 +66,5 @@ persistence_3_0.xjb
 persistence_3_0.xsd
 # excluded as this is a test file and we need to validate its content
 src/test/resources/static/classpath/index.html
+.hbs
```

Comment:
I am a bit surprised to hear you say that we support it since I added a new dependency to JKnack handlebars, and it seems I am the first to use anything in 3.x or 4.x for .hbs files in any other cases.

Fwiw, I tried to use the convention suggested and it failed the checkstyle check.  I will put it in my commit anyway for you to review.

Reply:
Follow-up will file bug on buildtools, and create a follow-up task here to remove .hbs exclusion.

Reply:
Logged https://github.com/helidon-io/helidon-build-tools/issues/829